rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -23,12 +23,10 @@ License: GPLv3
 URL: https://github.com/OpenSCAP/foreman_openscap
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
-Requires: foreman >= 1.8.0
+Requires: foreman >= 1.11.0
 
 Requires: scap-security-guide
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
-Requires: %{?scl_prefix}rubygem(scaptimony) >= 0.3.2
```

Comment:
Will Do. One question: When I `Obsoletes: %{?scl_prefix}rubygem(scaptimony)` I don't need to specify version number(s), right?